doc/content/getting-started/api/_index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Using the API"
+description: ""
+weight: 
+---
+
+While we recommend using the [Console]({{< ref "getting-started/console" >}}) or [CLI]({{< ref "getting-started/cli" >}}) to manage your applications and devices in {{% tts %}}, we also expose HTTP and gRPC APIs which you can interact directly with. This section contains information about using the HTTP API, and examples.
+
+<!--more-->
+
+A complete list of API endpoints is available in the [API Reference]({{< ref "reference/api" >}}). There, you can also find detailed information about [Authentication]({{< ref "reference/api/authentication" >}}) and [Field Masks]({{< ref "reference/api/field-mask" >}}).
+
+> If you're having trouble with the HTTP API, you can always inspect requests in the Console using your browser's inspector. All of the data displayed in the Console is pulled using HTTP API requests, and this should give you some insight in to how they are formed.
+
+## HTTP Queries
+
+Additional fields may be specified in HTTP requests by appending them as query string parameters. For example, to request the `name`, `description`, and `locations` of devices in an `EndDeviceRegistry.Get` request, add these fields to the `field_mask` field. To get this data for device `dev1` in application `app1`:
+
+```bash
+$ curl --location \
+  --header "Authorization: Bearer NNSXS.XXXXXXXXX" \
+  --header 'Accept: application/json' \
+  --header 'User-Agent: my-integration / my-integration-version' \
+  'https://thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations'
+```
+
+The same request in tenant `tenant1` on a multi-tenant deployment:
+
+```bash
+$ curl --location \
+  --header "Authorization: Bearer NNSXS.XXXXXXXXX" \
+  --header 'Accept: application/json' \
+  --header 'User-Agent: my-integration / my-integration-version' \
+  'https://tenant1.thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations'
+```
+
+Fields may also be specified as a JSON object in a POST request.
+
+To get a stream of events for device `dev1` in application `app1` :
+
+```bash
+$ curl --location \
+  --header 'Authorization: Bearer NNSXS.XXXXXXXXX' \
+  --header 'Accept: application/json' \
```

Comment:
The event stream accept header should be `Accept: text/event-stream`. See also the API reference: https://thethingsstack.io/reference/api/events/

Reply:
And maybe we should document what a `text/event-stream` response looks like. 

Background: https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events